parfive/downloader.py

```diff
@@ -102,6 +112,33 @@ def _generate_tokens(self):
             queue.put_nowait(Token(i + 1))
         return queue
 
+    @property
+    @lru_cache()
+    def use_aiofiles(self):
+        """
+        aiofiles will be used if installed and must be explicitly enabled
+
+        PARFIVE_OVERWRITE_ENABLE_AIOFILES takes precedence if present,
+        if the value is != "enable" aiofiles will not be used
+
+        finally the Downloader's constructure argument is considered.
+        """
+        if aiofiles is None:
+            return False
+
+        if "PARFIVE_OVERWRITE_ENABLE_AIOFILES" in os.environ:
+            return os.environ["PARFIVE_OVERWRITE_ENABLE_AIOFILES"] == "enable"
```

Comment:
For my other env vars I have just worked on if it's defined or not and ignored the value.